docs/Theming.md

```diff
@@ -69,3 +69,91 @@ code {
 ```
 
 This font provides built-in syntax highlighting for code blocks, making it easier to read and understand code snippets.
+
+## Theme Protocol (`browser://theme/`)
```

Comment:
This is great! Gotta remember to update the website docs with this too.

https://github.com/AgregoreWeb/website/blob/main/docs/theming.md

Reply:
https://github.com/AgregoreWeb/website/pull/74